src/Index.php

```diff
@@ -109,6 +110,19 @@ public function addGeoField(string $name, bool $noindex = false): IndexInterface
         return $this;
     }
 
+    /**
+     * @param string $name
+     * @param string $separator
+     * @param bool $sortable
+     * @param bool $noindex
+     * @return IndexInterface
+     */
+    public function addTagField(string $name, string $separator = ',', bool $sortable = false, bool $noindex = false): IndexInterface
```

Comment:
If SORTABLE and NOINDEX are supported for tag fields (see previous comment), I'd suggest putting $separator at the end of the argument list as it would probably be less often set to the non-default value.

Reply:
Done!